app/src/components/Grid.svelte

```diff
@@ -1,11 +1,46 @@
 <script>
   export let s = 17.73
+  let pixels = [];
   $: ss = s*5
 
   // reactive console -----------------------------------
   import bindConsole from '../lib/bindConsole'
   
-  window['grid'] = {}
+  window['grid'] = {
+    // maddie was heeerreee
+    // todo : migrate to a canvas api to make it faster (this is fine for now)
+    // from my testing, this can run a video (56x40) at 30fps (bad apple lol)
+    // it also handles screen capture pretty well (yes, doom was played on this)
```

Comment:
Love these other comments lmao, thanks for contributing, maybe we should finish this together? 🔥 I have code from another project that can fix the next step on the roadmap (implement propper screenspace to worldspace transform, right now everything is 100% screenspace with no concept of a "world")